web/models.py

```
@@ -2805,3 +2805,177 @@ class ScheduledPost(models.Model):
+        ("pending", "Pending Review"),
+        ("approved", "Approved"),
+        ("rejected", "Rejected"),
+    ]
+
+    applicant = models.ForeignKey(User, on_delete=models.CASCADE, related_name="chapter_applications")
+    chapter_name = models.CharField(max_length=200)
+    region = models.CharField(max_length=200)
+    description = models.TextField()
+    proposed_activities = models.TextField()
+    experience = models.TextField(
+        help_text="Applicant's experience with community organizing, education, or leadership"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)